src/palace/manager/sqlalchemy/model/integration.py

```diff
@@ -85,7 +85,7 @@ def context_update(self, new_context: dict[str, Any]) -> None:
     )
 
     # https://docs.sqlalchemy.org/en/14/orm/extensions/associationproxy.html#simplifying-association-objects
-    libraries: Mapped[list[Library]] = association_proxy(
+    library_associations: Mapped[list[Library]] = association_proxy(
```

Comment:
Yeah. I think we could. I was thinking that, in the short term, changing it would make it easier for me to find and replace names, which it did. But probably don't need to keep it, so I'll plan to pull that back out once I've finished the work from the library side and before I merge it.

Reply:
I've backed out the IC changes, so resolving this convo.